examples/replicated-kv/src/main.rs

```diff
@@ -38,7 +38,8 @@ async fn main() -> Result<()> {
         args.seeds.into_iter(),
     );
 
-    let node = DatacakeNodeBuilder::<DCAwareSelector>::new(1, connection_cfg)
+    let node_id: u8 = args.node_id.parse().unwrap();
```

Comment:
Nitpick: I think it'll be better to have the `node_id` type of the args struct be changed from a `String` to a `u8` as that way it'll prevent people accidentally providing a non-u8 ID and getting a slightly unhelpful error.

Very minor though

Reply:
I completely agree. In fact, I experimented with it different ways and this was actually my preferred solution, but I was uncertain if there was another implication.

I updated the code as requested and retested.